docs/src/Button.doc.js

```diff
@@ -9,281 +9,287 @@ import PageHeader from './components/PageHeader.js';
       },
       {
         name: 'inline',
         type: 'boolean',
+        required: false,
         defaultValue: false,
-        href: 'widths',
-      },
-      {
-        name: 'name',
-        type: 'string',
+        description:
+          'Display a button as an inline element so that it does not start on a new line breaking the flow of the content. Inline buttons are sized by the text within the button, whereas the default block buttons expand to the full width of their container.',
+        href: 'width',
       },
       {
         name: 'onClick',
         type: '({ event: SyntheticMouseEvent<> }) => void',
-      },
-      {
-        name: 'ref',
-        type: "React.Ref<'button'>",
-        description: 'Forward the ref to the underlying button element',
-        href: 'refExample',
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Callback fired when a button component is clicked (pressed and released) with a mouse or keyboard.',
+          'Required with button-type buttons.',
+        ]),
+        href: 'selected',
       },
       {
         name: 'size',
-        type: `"sm" | "md" | "lg"`,
+        type: `'sm' | 'md' | 'lg'`,
+        required: false,
         defaultValue: 'md',
-        description: 'sm: 32px, md: 40px, lg: 48px',
-        href: 'combinations',
+        description:
+          'Display a button in different sizes. Size only changes top & bottom padding modifying the component height: sm (32px), md (40px), lg (48px).',
+        href: 'height',
       },
       {
         name: 'text',
         type: 'string',
         required: true,
+        defaultValue: null,
+        description: buildDescription([
+          'Text to render inside the button to convey the function and purpose of the button. The button text has a fix size.',
+          'Accessibility: Screen readers read the `accessibilityLabel` prop, if present, instead of the `text` prop. See `accessibilityLabel` for more info.',
+        ]),
+        href: 'basic-button',
       },
       {
         name: 'textColor',
-        type: `"blue" | "red" | "darkGray" | "white"`,
-        description:
-          'The color for the text shown inside the Button. Note: should only be used for "white" buttons',
+        type: `'white' | 'darkGray' | 'blue' | 'red'`,
+        required: false,
+        defaultValue: null,
+        description: buildDescription([
+          'Primary colors to apply to the text inside the Button.',
+          'Text colors are automatically paired with the defined button `color`.',
+          '`textColor` should only be used when using a white background color; it may become necessary to manually override the text color to match that of the parent`s background. Do not override the text color for any button besides those with white background as it breaks design standards.',
+        ]),
         href: 'textColor',
       },
       {
         name: 'type',
```

Comment:
@AlbertCarreras thought a bit more about it over the weekend and I don't think we should switch on `type`. Mostly because we'll want a consistent API with `TapArea`.

What do you think about keeping `type` as is, with `button | submit` and introducing `role` with `button | link`?

```jsx
<TapArea role="link" href="https://pinterest.com" />
<Button role="link" href="https://pinterest.com" />
```

Reply:
DONE